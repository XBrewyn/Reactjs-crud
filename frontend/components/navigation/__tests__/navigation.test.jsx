import React from 'react';
import { create } from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navigation from '../navigation';

describe('<Navigation />', () => {
  it('renders correctly', () => {
    const tree = create(
      <Router>
        <Navigation />
      </Router>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
