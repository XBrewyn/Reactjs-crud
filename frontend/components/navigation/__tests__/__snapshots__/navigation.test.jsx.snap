// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> renders correctly 1`] = `
<ul
  className="css-13b5cob"
>
  <li>
    <a
      className="menu"
      href="/add"
      onClick={[Function]}
    >
      add
    </a>
  </li>
  <li>
    <a
      className="menu"
      href="/delete"
      onClick={[Function]}
    >
      delete
    </a>
  </li>
  <li>
    <a
      className="menu"
      href="/update"
      onClick={[Function]}
    >
      update
    </a>
  </li>
  <li>
    <a
      className="menu"
      href="/all"
      onClick={[Function]}
    >
      all
    </a>
  </li>
  <li>
    <a
      className="menu"
      href="/select"
      onClick={[Function]}
    >
      select
    </a>
  </li>
</ul>
`;
